client/src/constants/publisher_key.js

```diff
@@ -0,0 +1,5 @@
+const STRIPE_PUBLISHABLE = process.env.NODE_ENV === 'production'
+  ? 'pk_live_MY_PUBLISHABLE_KEY'
+  : 'pk_test_7mDiIF0ntVgLJjKz6B7evL0i00J88bDjzD';
```

Comment:
How do we test Stripe? Is there documentation we can look at to see how this can be tested?